libraries/ui-library/src/components/six-sidebar-item/six-sidebar-item.tsx

```diff
@@ -22,20 +22,25 @@ export class SixSidebarItem {
   /** Set to true to draw the sidebar item in a disabled state. */
   @Prop({ reflect: true }) disabled = false;
 
+  /** Provide if the item should be rendered as anchor tag. */
```

Comment:
We should probably mention, that the href is added automatically when using `routerLink` in Angular.

Reply:
done.